include/natalie/dsl.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "env.hpp"
+#include "global_env.hpp"
+#include "string.hpp"
+#include "symbol_value.hpp"
+#include "value_ptr.hpp"
+
+namespace Natalie {
+
+inline ValuePtr find_const(Env *env, SymbolValue *path) {
+    return GlobalEnv::the()->Object()->const_find(env, path);
+}
+
+inline ValuePtr find_nested_const(Env *env, std::initializer_list<SymbolValue *> paths) {
+    ValuePtr c;
+    for (auto path : paths) {
+        if (c == nullptr) {
+            c = find_const(env, path);
+            continue;
+        }
+        c = c->const_find(env, path);
+    }
+    return c;
+}
+
+}
```

Comment:
What are your thoughts on this `dsl.hpp` header? On first glance, I think these two new functions should go in our junk drawer known as `natalie.hpp`, but maybe you have thoughts about cleaning that up?

Reply:
Nah, it was in this header because the initial idea was a DSL to simplify the Interaction from c++ to the Natalie runtime, but that wasn't needed so I'd rather move it to the Natalie header and if we ever think it needs to be cleaned up we will :)